icu4c/source/common/unicode/locid.h

```diff
@@ -523,6 +577,54 @@ class U_COMMON_API Locale : public UObject {
      */
     void setKeywordValue(const char* keywordName, const char* keywordValue, UErrorCode &status);
 
+#ifndef U_HIDE_DRAFT_API
+    /**
+     * Sets or removes the value for a keyword.
```

Comment:
On both the existing and new keyword value setters, please add documentation as discussed in the API discussion.
https://docs.google.com/document/d/15r3s4ZUFw1crAK44VHi6UPbvrXQca9b-nFRCyFKKzHc/edit#heading=h.dwuzemgnou2m

"set[Unicode]KeywordValue() document no mapping vs. accept Unicode names/values"